core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -646,6 +650,17 @@ private void filterAndAddTopicRequest(
     }
   }
 
+  private void filterAndAddOperationalRequest(
+      List<OperationalRequest> operationalRequestList,
+      OperationalRequest row,
+      String wildcardSearch,
+      boolean applyWildcardFilter) {
+    if (!applyWildcardFilter
+        || row.getTopicname().toLowerCase().contains(wildcardSearch.toLowerCase())) {
```

Comment:
```suggestion
        || row.getTopicname().toLowerCase(Locale.ROOT).contains(wildcardSearch.toLowerCase(Locale.ROOT))) {
```
It's better to specify charset parameter for `toLowerCase` otherwise for some charsets results could be unpredictable